Discovery/Web-Content/api/README.md

```diff
@@ -9,6 +9,7 @@ A wordlist of API names used for fuzzing web application APIs.
 * actions-lowercase.txt - API function name verbs with leading character lower-case
 * objects-uppercase.txt - API function name nouns with leading character upper-case
 * objects-lowercase.txt - API function name nouns with leading character lower-case
+* api-endpoints-res.txt - Resultuent file for all API files
```

Comment:
`Resultuent` should be `Resultant` :p
But besides that, I think it would be more clear if it said "Combination of all of the files above (uniq'ed)"

Reply:
Oops, My bad

Resolved :)

Thanks.